src/app/public/modules/overlay/overlay.component.html

```diff
@@ -0,0 +1,19 @@
+<div
+  class="sky-overlay"
```

Comment:
For the overlay itself, we probably don't need to include ARIA attributes since it's more or less a visual element. The content within the overlay will definitely need ARIA attributes (such as the modal), but that would be up to the component being injected into the overlay.